scripts/links.py

```diff
@@ -27,14 +24,14 @@ def should_filter(url):
         return url.startswith("mailto") or url in _filtered
 
     for file in files:
-        with open(file, "rb") as f:
-            content = BytesParser(policy=policy.default.clone(utf8=True)).parse(f)
-            selector = urlchoose.URLChooser(
-                urlscan.msgurls(content), dedupe=False, reverse=False, shorten=False
-            )
-            yield from zip(
-                repeat(file), filter(lambda url: not should_filter(url), selector.urls)
-            )
+        cmd = ['python', '-m', 'urlscan', '-n', f'{file}']
+        r = subprocess.run(cmd, stderr=subprocess.PIPE, stdout=subprocess.PIPE)
+        r.check_returncode()
+        stdout = r.stdout.decode('utf8').strip()
```

Comment:
Hm, I believe you hide the stderr of the subprocess here. This will make debuging issues difficult...

Reply:
> Hm, I believe you hide the stderr of the subprocess here. This will make debuging issues difficult...

Yes you are right, stderr won't be written to stderr but instead will be part of the result object r.
This is intentional though, I don't want outside observers to see the implementation detail(s) of this function
(That the urls now retrieved via the cli intertface/command). If the cli call fails all information will be available in
the exception and the result object .